README.md

```diff
@@ -522,8 +524,12 @@ and not directly from Python (otherwise it may not propagate to child processes)
 
 **2. Metrics collector**:
 
-The application must initialize a new `CollectorRegistry`,
-and store the multi-process collector inside.
+The application must initialize a new `CollectorRegistry`, and store the
+multi-process collector inside. It is a best practice to create this registry
+inside the context of a request to avoid metrics registering themselves to a
+collector used by a `MultiProcessCollector`. If a registry with metrics
+registered is used by a `MultiProcessCollector` duplicate metrics may be
+exported, one for multiprocess, and one for the process serving the request.
```

Comment:
i would add an example Counter metric to the below snippet, that does not use registry=

Reply:
:+1: I added one.